test/e2e/cluster_test.go

```diff
@@ -130,7 +130,7 @@ var _ = Describe("Workload cluster creation", func() {
 		}, result)
```

Comment:
can we add an e2e test which we can run but disabled by default? that will help quickly test out any further changes. We can create. anew template and e2e test case

Reply:
- [ ] Add a new regional client e2e test as well as create new images for the region to test against